environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentApiConverter.java

```diff
@@ -326,4 +328,11 @@ public EnvironmentFeatures convertToEnvironmentTelemetryFeatures(FeaturesRequest
         features.setClusterLogsCollection(featuresRequest.getClusterLogsCollection());
         return features;
     }
+
+    public EnvironmentLoadBalancerDto initLoadBalancerDto(EnvironmentLoadBalancerUpdateRequest request) {
```

Comment:
**suggestion:** You might consider a `.from()` or `.toDto()` static method on the `EnvironmentLoadBalancerDto` or `EnvironmentLoadBalancerUpdateRequest` respectively.

Reply:
I generally try to avoid static methods in Cloudbreak. I know they're used some places in the code, but generally I find the code easier to work with when the methods aren't static.

Reply:
I say that and realize `builder()` is a static method :). Regardless, I understand your point, but prefer to follow existing patterns unless it's a chore to.